DSCResources/MSFT_xADRecycleBin/Docs/readme.md

```diff
@@ -0,0 +1,15 @@
+The xADRecycleBin DSC resource will enable the Active Directory Recycle Bin feature for the target forest.
```

Comment:
This information should go to the root [README.md](https://github.com/PowerShell/xActiveDirectory/blob/master/README.md). I didn't fully convert html documentation to markdown and I guess this was the reason of the confusion. I will update README.md today, so you can include this section.